packages/design-system/src/components/Stack/Primitive/StackPrimitive.tsx

```diff
@@ -38,7 +38,7 @@ export const sizeOptions = {
 	XL: 'XL',
 };
 
-export const heightOptions = {
+export const sizeOptions = {
```

Comment:
isn't a breaking change ?

Reply:
I don't think it's a breaking change as i dough the fact it's meant to be use in the host project 👀

Reply:
It's not exported in the lib yeah